bodhi/server/templates/update.html

```diff
@@ -138,7 +142,7 @@
         '<td>' + icon + '</td>' +
         '<td class="nowrap">' + testcase + '</td>' +
         '<td class="stretch-table-column text-xs-right">' + note + '</td>' +
-        '<td class="nowrap">' + age + '</td>' +
+        '<td class="nowrap">' + age + '</td>' + '<td>' + rerun + '<td/>' +
```

Comment:
Like you, I have no idea how to make this work off the top of my head. Let's hope that @ryanlerch can save us!